### ISSUE TITLE
Orphis buildbot staled since switch to git

### ISSUE THREAD

**Issue by pal1000** [gid15/jpcsp#381]
I'd inform Orphis myself but the e-mail address he posted on his website is obfuscated in a fashion that beats me. I don't feel like doing bulk of shots in the dark in an attempt to reach the correct e-mail address. Not to mention this could be interpreted as an spam attack.

**Comment by hlide** [comment]
A message was sent to him.

**Comment by pal1000** [comment]
I'll close when I see it working again, if not someone beats me to it.

**Comment by gid15** [comment]
I've e-mailed Orphis a few days ago, but got no feedback yet.

**Comment by hlide** [comment]
I had Orphis yesterday, he told me he would do something. I thought it was already done.

**Comment by sum2012** [comment]
@gid15 have commit this
https://github.com/gid15/jpcsp/commit/8d10998a784ccac3be6b7c8ca3f9a9b372df3fea
but @Orphis  buildbot no change

**Comment by DragonNeos** [comment]
@gid15 Any word from @Orphis yet? It would be nice to have an alternative source to be able to download builds since RapidGator isn't exactly user friendly (having a download restriction of 1 file per 120 minutes for a Slow Speed Download).

**Comment by vgturtle127** [comment]
@pal1000 @hlide @gid15 @sum2012 @Orphis @DragonNeos Has any work been done on this or any other solution?

RapidGator royally sucks the big ones, it only allows an extremely slow download, and it usually takes hours for it to let you download another file. For testing multiple versions of the same emulator, or especially the software renderer plugin which has multiple versions for CPU architecture AND CPU instructions, this is absolutely horrid. I'd much rather just download one big archive and wait longer for it to download, but on the slow RapidGator site, that might take longer than it takes to just wait for the next stable release (even though I've never heard of a stable release of JPCSP, still might take less time).

If Orphis can supply a buildbot, that would be great. Any alternative would be greatly appreciated, and is much needed in my opinion.

**Comment by pal1000** [comment]
Yes, it would be a good thing for software renderer to be added to the buildbot. Waiting an 1h between downloads is really bad.

**Comment by Margen67** [comment]
@Orphis
